index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="sv">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="./lucky.style.css">
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+ <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Abyssinica+SIL&family=Roboto+Condensed:ital@1&display=swap" rel="stylesheet">
+ <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+ <title>Lucky Duck</title>
+</head>
+<body>
+ <div class="Navbar">
```

Comment:
I think the height of the navbar is a bit too large

Reply:
I will make it smaller

Reply:
I believe the height is good now. Thanks!